### PR TITLE
add plugin version to contribution-guide

### PR DESCRIPTION
**Pull request by urisilberstein**
@aviemzur please review this, thanks

### CODE REVIEW

**Comment by urisilberstein** [comment]
@aviemzur I've updated PR

File: src/contribute/contribution-guide.md [hunk 1]
```diff
@@ -122,7 +122,10 @@ To configure annotation processing in IntelliJ:
 ###### Checkstyle
 IntelliJ supports checkstyle within the IDE using the Checkstyle-IDEA plugin.
 
-1. Install the "Checkstyle-IDEA" plugin from the IntelliJ plugin repository.
+1. Install the "Checkstyle-IDEA" plugin
+    1. download version 5.3.1 from [Checkstyle-IDEA versions](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea)
```
Comment (aviemzur) [inline src/contribute/contribution-guide.md]: First letter in sentence should be capitalized. (Same for next couple of sentences.

**Comment by kennknowles** [comment]
Now you've got some conflicts, so best to rebase the PR. You can squash the two commits together.

**Comment by aviemzur** [comment]
Can this PR be closed?

**Pull request closed without merging**